Add summary for restart vm after force delete vm (#376)
harvester/docs: docs/advanced/settings.md

Default: `{"enable":true, "period":300}`

:::note

> When the host has gone or powered off, the VM will only reboot and not migrate.

Reviewer: ```suggestion
When a host is unavailable or is powered off, the VM only reboots and does not migrate.
```
